README.md

```diff
@@ -1,34 +1,159 @@
-# Git Repository Template
+This NimbleLoyalty SDK library allows developers to easily integrate loyalty programs into their iOS applications.
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+- [Features](#features)
+- [Component Libraries](#component-libraries)
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Credits](#credits)
+- [License](#license)
+
+## Features
+
+- Authenticate (via Auth0)
+- Get reward list
+- Redeem reward
+- Get reward history
+
+## Component Library
+
+- Moya
+- KeychainAccess
+
+## Requirements
+
+- iOS 14.0+
+- Xcode 14.0+
+
+## Installation
+
+The iOS Loyalty SDK can be installed using CocoaPods or Swift Package Manager.
+
+### CocoaPods
+
+To install using CocoaPods, add the following line to your Podfile:
+
+```
+pod 'NimbleLoyalty', '~> 0.1.0'
+```
+
+### Swift Package Manager
+
+To install using Swift Package Manager, add the following line to your dependencies:
+
+```swift
+.package(url: "https://github.com/nimblehq/business-loyalty-ios-sdk-poc.git", from: "0.1.0")
+```
 
 ## Usage
 
-Clone the repository
+#### Authentication
 
-`git clone git@github.com:nimblehq/git-template.git`
+Before authenticating the user, remember to set the **clientId** and **clientSecret** (optional).
```

Comment:
Why we have `optional` here?

Reply:
Because clientSecret is not compulsory while authenticating, it can be empty ^^